constants_local.xml

```diff
@@ -16,15 +16,17 @@
 		<POW2_DIFFICULTY>3</POW2_DIFFICULTY>
 		<MICROBLOCK_TIMEOUT>90</MICROBLOCK_TIMEOUT>
 		<VIEWCHANGE_TIME>180</VIEWCHANGE_TIME>
+        <VIEWCHANGE_EXTRA_TIME>10</VIEWCHANGE_EXTRA_TIME>
 		<SHARDING_TIMEOUT>180</SHARDING_TIMEOUT>
 		<CONSENSUS_OBJECT_TIMEOUT>10</CONSENSUS_OBJECT_TIMEOUT>
 		<FINALBLOCK_CONSENSUS_OBJECT_TIMEOUT>30</FINALBLOCK_CONSENSUS_OBJECT_TIMEOUT>
 		<NUM_FINAL_BLOCK_PER_POW>5</NUM_FINAL_BLOCK_PER_POW>
 		<NUM_DS_KEEP_TX_BODY>5</NUM_DS_KEEP_TX_BODY>
-		<MAXMESSAGE>64</MAXMESSAGE>
+		<MAXMESSAGE>32</MAXMESSAGE>
 		<MAXSUBMITTXNPERNODE>10</MAXSUBMITTXNPERNODE>
```

Comment:
Why need to reduce MAXMESSAGE? If needed to do so, make sure to sync to testnet.

Reply:
I noticed some of the user observed out of resource issue on slower machine. Hence, I have advise them to change to a lower number. There is no noticeable impact. 

Good point though. I need to check other scripts too.